spec/spec_helper.rb

```diff
@@ -28,6 +28,7 @@
 
     # recipes/repository.rb
     stub_command('rpm -q gpg-pubkey-e09422b3').and_return(false)
+    stub_command('rpm -q gpg-pubkey-20200908').and_return(false)
```

Comment:
20200908 is the key's creation date (we switched to naming key files by creation date because it'll be easier to follow in the future). The short fingerprint of the key is `fd4bf915`, thus the correct command is `rpm -q gpg-pubkey-fd4bf915`:

```suggestion
    stub_command('rpm -q gpg-pubkey-fd4bf915').and_return(false)
```